src/Simulator/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ SET (HDRS
 	VaporizeFluidSimulator.h
 )
 
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+	set_source_files_properties(FireSimulator.cpp PROPERTIES COMPILE_FLAGS "-Wno-error=global-constructors")
+endif()
```

Comment:
I disabled the warnings this way because it was what I thought most of the other code used to disable unwanted warnings. In any case, I don't have any experience using pragmas to disable warnings, could you link another place in the code where they are used so I can see what I need to do?

Reply:
Some of the code does it this way, some with pragmas.  If you look at #4013 though, you see lots of warnings disabled this way become stale.
Here's an example of doing it with pragmas:
https://github.com/cuberite/cuberite/blob/aff140365de993602724f7324e78b6283bf71155/src/Bindings/LuaState.cpp#L2031-L2040